src/utils/transactions/getAddressLink.js

```diff
@@ -2,8 +2,10 @@
 
 import config from 'config'
 
-function getAddressLink(address: Address, blockExplorerSubdomain: string): string {
-  return `//${blockExplorerSubdomain}.${config.blockExplorerLink}/address/${address}`
+function getAddressLink(address: ?Address, blockExplorerSubdomain: string): string {
+  return address
+    ? `//${blockExplorerSubdomain}.${config.blockExplorerLink}/address/${address}`
+    : ''
```

Comment:
I think, we can get strict `Address` type as parameter without nullable state, and check nullable state of parameter before call this function.